kuma/wiki/views/misc.py

```diff
@@ -42,6 +42,7 @@ def ckeditor_config(request):
                   content_type='application/x-javascript')
 
 
+@block_user_agents
```

Comment:
Should we add https://developer.mozilla.org/en-US/docs/get-documents to robots.txt?